modules/kmm-building-in-cluster.adoc

```diff
@@ -45,3 +45,5 @@ Otherwise, KMM creates a `Build` resource to build your image. After the image i
 <6> Required.
 <7> Optional: Avoid using this parameter. If set to `true`, KMM will be allowed to check if the container image already exists using plain HTTP.
 <8> Optional: Avoid using this parameter. If set to `true`, KMM will skip any TLS server certificate validation when checking if the container image already exists.
+
+Successful build pods are garbage-collected immediately, unless [`job.gcDelay`](./configure.md#jobgcdelay) is set in the Operator configuration. Failed build pods are always preserved and must be deleted manually by the administrator for the build to be restarted.
```

Comment:
```markdown
[`job.gcDelay`](./configure.md#jobgcdelay)
```
is not a valid link in asciidoc.

Reply:
@qbarrand Should this be changed to just `job.gcDelay`?

Reply:
In the upstream docs, this links to the table listing all operator settings. Could we replicate that in the product docs?

Reply:
The problem with doing that is Red Hat style guidelines. I could add the link under `Additional resources` but it may be rejectedd in Peer review. 
![image](https://github.com/openshift/openshift-docs/assets/5238708/85030b1e-ff00-40ee-826f-c247fcbc3515)

Reply:
Then let us stick to just `job.gcDelay` indeed.